src/Features/Activity/Activity.tsx

```diff
@@ -81,6 +85,23 @@ function WorkflowActivity() {
   const activitySummaryState = useQuery(activitySummaryUrl);
   const activityStatusSummaryState = useQuery(activityStatusSummaryUrl);
   const activityState = useQuery(activityUrl);
+
+  const systemUrl = serviceUrl.getSystemWorkflows();
+
+  const {
```

Comment:
Minor comment here but I have been favoring not destructuring the query object and referencing the property like `systemWorkflows.isLoading` , `systemWorkflows.error` etc recently. I don't care what you do but I do find it to be easier to read.